src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -78,8 +97,10 @@ private async Task<string> GetBundle(HttpClient httpClient)
                     return null;
                 }
 
+                _extensionBundleVersion = latestBundleVersion;
```

Comment:
There are multiple code paths setting this variable. Can we reconcile things here so that isn't the case? Looks like we have some duplication of logic across some of these methods.